linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala

```diff
@@ -40,9 +40,10 @@ trait Router {
 
   protected def configureServer(s: Server): Server
 
-  def withParams(ps: Stack.Params): Router =
-    _withParams(ps)
-      .withServers(servers.map(configureServer(_)))
+  def withParams(ps: Stack.Params): Router = {
+    val routerWithParams = _withParams(ps)
+    routerWithParams.withServers(routerWithParams.servers.map(routerWithParams.configureServer))
+  }
```

Comment:
this is a bugfix right? not worth mentioning in changes, but worth mentioning in commit